cli/src/main/java/org/jboss/as/cli/handlers/OperationRequestHandler.java

```diff
@@ -184,18 +184,13 @@ private ModelNode validateRequest(CommandContext ctx, ModelNode request) throws
                     throw new CommandFormatException("Operation '" + operationName + "' does not expect any property.");
                 }
             }
```

Comment:
I will double check but this commit should have reverted everything back to how it was before